activemount.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/go-plugins-helpers/volume"
+)
+
+type ActiveMount struct {
```

Comment:
You probably meant to call it `activeMount` (see point 2 of the next note, on exporting entities in go).
(note that unlike the structure, its field _should_ be exported: otherwise `json.Marshal`/`.Unmarshal` won't be able to access it, as they are not part of this package)

Reply:
It's my first real program in Go, I had no idea about that, I simply copied how `VolumeInfo` was defined. I have fixed it.